Apps/WebClient/src/ClientApp/app/models/medicationStatement.ts

```diff
@@ -0,0 +1,25 @@
+// Medication statement model
+export default class MedicationStatement {
+  // Drug Identification Number for the medication statement.
+  public DIN!: string;
+  // Brand name of the medication statement.
+  public brandName!: string;
+  // Common or generic name of the medication statement.
+  public genericName!: string;
+  // Quantity for the medication statement.
+  public quantity!: number;
+  // Medication statement dosage.
+  public dosage!: number;
+  // Medication statement prescription status.
+  public prescriptionStatus!: string;
+  // Date the medication statement was dispensed.
+  public dispensedDate!: Date;
+  // Surname of the Practitioner who issued the medication statement.
+  public practitionerSurname!: string;
+  // Drug medication discontinued date, if applicable.
+  public drugDiscontinuedDate!: Date;
```

Comment:
this is a nullable field (change from ! to ?)

Reply:
changed